client/packages/common/src/ui/layout/tables/utils/ColumnUtils.ts

```diff
@@ -54,10 +54,10 @@ export const useColumnUtils = () => {
           return !!obj ? [obj] : [];
         });
         const isValid =
-          arr.length === 0 ||
-          arr[0] === undefined ||
-          typeof arr[0] !== 'object' ||
-          !(key in arr[0]);
+          arr.length > 0 &&
+          (arr[0] === undefined ||
+            typeof arr[0] !== 'object' ||
+            !(key in arr[0]));
```

Comment:
@mark-prins I wasn't sure about the other `isValid` conditions and why they are there. Do you have more info on it? then I will add it as a comment...

Tests are still passing with change...

Reply:
not sure sorry!
Would have been taken from the previous implementation.. but makes little sense reading it now 🙄 
I would do this if I was looking at it, which is pretty messy and of course I don't know if it matches the original intention

```
        const isValid =
          arr.length > 0 &&
          !!propertyKey &&
          arr[0] !== undefined &&
          typeof arr[0] === 'object' &&
          propertyKey in arr[0];
```